DIPs/DIPxxxx.md

```diff
@@ -0,0 +1,233 @@
+# Enhanced Attribute Defaults
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | anass-O                                                         |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Draft                                                           |
+
+## Abstract
+
+This DIP proposes the following two changes:
+
+* Introducing new attributes `throw`, `impure`, and `@gc` as inverses of `nothrow`, `pure`, and `@nogc`.
+
+* Allowing attributes (`@safe`, `@trusted`, `@system`, `throw/nothrow`, `pure/impure`, `@gc/@nogc`) that mutate function behavior to be set at the module level.
+
+
+## Contents
+* [Rationale](#rationale)
+* [Prior Work](#prior-work)
+* [Description](#description)
+* [Breaking Changes and Deprecations](#breaking-changes-and-deprecations)
+* [Reference](#reference)
+* [Copyright & License](#copyright--license)
+* [Reviews](#reviews)
+
+## Rationale
+
+Currently there is no easy way to set a default for function attributes. This includes the attributes for determining memory safety (`@safe`, `@trusted`, and `@system`) as well as the attributes `nothrow`, `pure`, and `@nogc`; which have varying affects.
```

Comment:
> which have varying affects

I read that as "various effects", if I take "varying affects" literally I don't know what you mean.